### DISCUSSION TITLE
Scoopable/Fuel Star Icon or Marker on Navigation List

### DISCUSSION THREAD

**Discussion by alyyasser** [iaincollins/icarus#10] [Ideas & Suggestions]
It would nice to have fuel star/scoopable icon on the star shown on system list

**Comment by iaincollins** [comment]
I'd thought about this but couldn't think of an icon to use for it!

The in-game icon for fuel scoops is odd looking and hard to make out when small, and a petrol pump handle or gas canister seemed maybe out of place (but still, might be helpfully obvious?).

Open to ideas!

**Comment by alyyasser** [comment]
petrol pump handle feels right for me, it's the right icon/indicator that felt familiar and obvious. or maybe the icon used in car dashboard?
im really not good at designing things (even though i am a mobile app/frontend dev myself)
